spec/notify/notify_despatchers/notify_spec.rb

```diff
@@ -185,17 +182,37 @@ def personalisation
       }
     end
   end
+  let(:notification) { subject.new to: recipients, name: 'Test User' }
 
   before { allow(Feature).to receive(:active?).with(:sms) { true } }
 
   subject { StubSmsNotification }
+
   include_examples "notify_client"
 
   context "when in non-production environments" do
     before { allow(Feature).to receive(:active?).with(:sms) { false } }
 
     it "does not despatch SMS" do
+      perform_enqueued_jobs do
+        notification.despatch_later!
+      end
+
       expect(NotifyService.instance).not_to have_received(notify_method)
     end
   end
+
+  context "when there is a non-mobile number" do
+    let(:valid_recipients) { %w[07777777778 07777777779] }
+    let(:invalid_recipients) { %w[12345678910] }
```

Comment:
Parsing this number will probably return invalid number. Can we also test against a valid landline number?